chains/rsktestnet/metadata.yaml

```diff
@@ -0,0 +1,28 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://rootstock-testnet.blockscout.com/api
+    family: blockscout
+    name: Blockscout
+    url: https://rootstock-testnet.blockscout.com
+blocks:
+  confirmations: 1
+  estimateBlockTime: 30
+  reorgPeriod: 4
+chainId: 31
+deployer:
+  name: Rootstock
+  url: https://rootstock.io
+displayName: Rootstock Testnet
+domainId: 31
+isTestnet: true
+name: rootstocktestnet
```

Comment:
The folder name must match this string, so `rootstocktestnet`

Reply:
Fixed! Thanks for pointing it out.